From 46761b7fdb0707036482c98eb3a80279753df484 Mon Sep 17 00:00:00 2001
Message-Id: <46761b7fdb0707036482c98eb3a80279753df484@dist-git>
From: Jiri Denemark <jdenemar@redhat.com>
Date: Mon, 24 Apr 2017 15:38:41 +0200
Subject: [PATCH] conf: Rename mode parameter in virCPUDefParseXML

The type of this parameter is virCPUType so calling it 'mode' is pretty
strange, 'type' is a much better name.

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
(cherry picked from commit e841a41169dd726af13f29cb57fbafe658b7011a)

https://bugzilla.redhat.com/show_bug.cgi?id=1428952

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/conf/cpu_conf.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/conf/cpu_conf.c b/src/conf/cpu_conf.c
index b78531e60..623b1699f 100644
--- a/src/conf/cpu_conf.c
+++ b/src/conf/cpu_conf.c
@@ -230,7 +230,7 @@ virCPUDefCopy(const virCPUDef *cpu)
 virCPUDefPtr
 virCPUDefParseXML(xmlNodePtr node,
                   xmlXPathContextPtr ctxt,
-                  virCPUType mode)
+                  virCPUType type)
 {
     virCPUDefPtr def;
     xmlNodePtr *nodes = NULL;
@@ -250,7 +250,7 @@ virCPUDefParseXML(xmlNodePtr node,
     if (VIR_ALLOC(def) < 0)
         return NULL;
 
-    if (mode == VIR_CPU_TYPE_AUTO) {
+    if (type == VIR_CPU_TYPE_AUTO) {
         if (virXPathBoolean("boolean(./arch)", ctxt)) {
             if (virXPathBoolean("boolean(./@match)", ctxt)) {
                 virReportError(VIR_ERR_XML_ERROR, "%s",
@@ -263,7 +263,7 @@ virCPUDefParseXML(xmlNodePtr node,
             def->type = VIR_CPU_TYPE_GUEST;
         }
     } else {
-        def->type = mode;
+        def->type = type;
     }
 
     if ((cpuMode = virXMLPropString(node, "mode"))) {
-- 
2.12.2

